pepper/test/apps/isBoolVerification_test.cpp

```diff
@@ -0,0 +1,32 @@
+#include <cassert>
+#include <gtest/gtest.h>
+#include "declarations.h"
+#include <apps/hash_transform.h>
+
+#include <apps/dex_common.h>
+#include "util.h"
+
+struct TestStructArray { field digest[256]; };
+
+TEST(isBoolVerification, isBoolVerificationWithBools) { 
```

Comment:
Ideally this wouldn't be part of this pull request (it's kind of unrelated).

Reply:
removed it